codes/kotlin/chapter_tree/avl_tree.kt

```diff
@@ -144,11 +143,11 @@ class AVLTree {
             } else {
                 // 子节点数量 = 2 ，则将中序遍历的下个节点删除，并用该节点替换当前节点
                 var temp = node.right
-                while (temp!!.left != null) {
-                    temp = temp.left
+                temp?.left?.let {
```

Comment:
Is it equivalent?

Reply:
The output is same, In this situation.

Reply:
Maybe is inappropriate, I will change it back.

Reply:
The same outputs do not mean they're equivalent. There is no `while` loop in the new code.

Reply:
I confirm, it`s been changed.